bin/cml/runner.js

```diff
@@ -401,6 +401,12 @@ const run = async (opts) => {
     }
   }
 
+  if (driver === 'github') {
+    winston.warn(
+      'Github Actions timeout has been updated from 72h to 35 days. Update your workflow accordingly to be able to restart it automatically.'
+    );
```

Comment:
can we do better? e.g. `grep 'timeout-minutes: 4320' .github/workflows/*.y*ml && raise`?

Reply:
Instead of `throw` maybe we can emit a warning to GitHub's user interface.

E.g. `console.log('::warning GitHub Actions timeout has been...')`

Reply:
I'd favour visibility (`throw`) or silent correction (`const GH_5_MIN_TIMEOUT = (timeout_minutes - 5) * 60 * 1000`)

Reply:
Then, let's `throw`

Reply:
opened #1097 :)